src/deion/ui/views/GodMode.js

```diff
@@ -735,6 +746,37 @@ class GodMode extends React.Component {
 								value={this.state.brotherRate}
 							/>
 						</div>
+						<div className="col-sm-3 col-6 form-group">
```

Comment:
Make these options only show for Basketball GM, not Football GM. Look at how that's done already for the All-Star Game option.